### ISSUE TITLE
OS Application does not launch in Windows 10

### ISSUE THREAD

**Issue by svpxdev** [openstudiocoalition/OpenStudioApplication#400]
**Issue**
I installed the latest Open Studio Application and I am not able to start the application. On clicking the application, I see a small window open up and then disappear with no further action or even a running thread in the task manager. 

**Please see the video [here]( https://vimeo.com/574899954/76ea3ef5ff)**

**What I have tried:**
- Reinstall application
- Restart after uninstall and install
- Checked for compatibility by running in compatibility mode (Not sure if that ever helps, but tried it anyways)
- Checked video card drivers - as suggested in one of the threads having similar issue
- Checked if Dell - Back and Update application was causing an issue - It wasn't, since its not installed on the system.
- Installed previous version of OSA, similar behavior was observed. 

**System and Environment Info**
- Dell XPS 15 7590
- Windows 10 Pro 21H1
- OS Build 19043.1052 / Windows Feature Experience Pack 120.2212.2020.0
- Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz   2.40 GHz
- 64-bit operating system, x64-based processor
- 16GB RAM
- Nvidia GeForce GTX 1650

**Comment by macumber** [comment]
Thanks for the detailed report.  Can you open a command prompt (type `cmd` then return in the Windows search bar), then give the following commands:

```
cd c:\openstudioapplication-1.2.0\bin
set OPENSTUDIO_APPLICATION_DEBUG=1
set OPENSTUDIO_APPLICATION_LOGFILE_PATH=%HOMEDRIVE%%HOMEPATH%\Desktop\osapp.log
.\OpenStudioApp.exe
```

![image](https://user-images.githubusercontent.com/1327850/125728620-e23dd40d-d2d6-4513-bb83-fe70e24a544f.png)

That should produce a `osapp.log` file on your Desktop, could you upload it here?

**Comment by svpxdev** [comment]
Thanks for the response and sorry for my late reply, I was on vacation. 

Here is the log file created after running the commands. 

[osapp.log](https://github.com/openstudiocoalition/OpenStudioApplication/files/6846137/osapp.log)

**Comment by macumber** [comment]
Thanks for the log.  There are some pretty unusual items in there, did you happen to build the OS App yourself?  I don't know any other reason why it would be referring to a an IddFile at `D:\OSN\src\utilities\idd\IddField.cpp`.  It seems like this was a custom build that was built against an invalid version of the OS SDK?

**Comment by svpxdev** [comment]
No, not really. I just installed it from the windows installer. Did not see a reason to build it my self since I have used the installer before and it has always worked. It's the first time I have had this issue.

**Comment by macumber** [comment]
Can you uninstall the version that you have and try reinstalling one of the installers from this official release:

https://github.com/openstudiocoalition/OpenStudioApplication/releases/tag/v1.2.0

**Comment by svpxdev** [comment]
Hi, 
So here is what I did now, uninstalled OSA and reinstalled from the link you have provided, still seem to get the same error. I have attached a **[screen recording](https://vimeo.com/577477624/555a9f179a)** with this post. Am I missing a step or is it more system related ? 

Also attaching the log once again. 
[osapp.log](https://github.com/openstudiocoalition/OpenStudioApplication/files/6853184/osapp.log)

**PS:** Restarting does not help.

**Comment by macumber** [comment]
@jmarrec you have any idea what the path `D:\OSN\src\utilities\idd\IddField.cpp` could be from?

**Comment by macumber** [comment]
@svpxdev could you try opening the file explorer and going to your new install (should be `C:\openstudioapplication-1.2.0\bin`) and double clicking the `OpenStudioApp.exe` file there?  I wonder if that shortcut is pointing to some other installed version?

**Comment by svpxdev** [comment]
There is just one version of installed OSA
![image](https://user-images.githubusercontent.com/71379971/126613139-81d34a85-e92d-485a-a367-768cd26685d6.png)

And has the same behaviour
![image](https://user-images.githubusercontent.com/71379971/126613191-fd9c311c-118f-4061-a7c7-c2f4c9ab4341.png)

**Comment by jmarrec** [comment]
I think that `D:\OSN` might just be the path of the build dir on the Jenkins CI instance that built the OS SDK. 

(I'm a bit confused by why the CLI (`openstudio`) is from 04/05 but `openstudiolib.dll` is from 08/06, but that's probably just cmake changing the timestamp on the dll)

From the log, it seems that the SDK doesn't manage to read its own IDD, or an older version of the IDD. I would start by removing any user defined Libraries... This is very annoying on windows when the GUI doesn't work, but you can do it by launching the registry editor. Do Windows Key + R, then type `regedit`

Go to `Computer\HKEY_CURRENT_USER\SOFTWARE\NREL\OpenStudioApp\library\` and delete the `library` entry altogether (right click on the left panel, then click "Delete"). Before, you do you, could you tell us how many entries you have in there and what they point to? Eg: I have 3 entries, 1 is hvac_library.osm, two is office_default.osm and the third is a dummy file I added at C:\Untitled.osm

![image](https://user-images.githubusercontent.com/5479063/126764013-6f2be768-c9b4-4b71-9a62-17b36bddd14a.png)

Eg: first entry:

![image](https://user-images.githubusercontent.com/5479063/126764254-0dfdd2f1-af5d-4420-b7ea-9ae842fd3b45.png)

**Comment by svpxdev** [comment]
I dont have the library installed it seems like. 
![image](https://user-images.githubusercontent.com/71379971/126985944-84c822cb-3638-42f3-83b0-b890958aec95.png)
